kv/txn_coord_sender.go

```diff
@@ -288,120 +291,101 @@ func (tc *TxnCoordSender) startStats() {
 // Send implements the client.Sender interface. If the call is part
 // of a transaction, the coordinator will initialize the transaction
 // if it's not nil but has an empty ID.
+// TODO(tschottdorf): move to batch.Sender interface.
```

Comment:
err, nevermind. probably just want to change this to "TODO(tschottdorf): remove this method in favour of SendBatch"